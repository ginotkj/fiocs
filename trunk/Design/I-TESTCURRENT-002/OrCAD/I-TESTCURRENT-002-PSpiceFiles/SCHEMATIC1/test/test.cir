** Profile: "SCHEMATIC1-test"  [ D:\Documents\TESIS\fiocs\Design\I-TESTCURRENT-002\OrCAD\i-testcurrent-002-pspicefiles\schematic1\test.sim ] 

** Creating circuit file "test.cir" 
** WARNING: THIS AUTOMATICALLY GENERATED FILE MAY BE OVERWRITTEN BY SUBSEQUENT SIMULATIONS

*Libraries: 
* Profile Libraries :
* Local Libraries :
* From [PSPICE NETLIST] section of C:\OrCAD\OrCAD_16.0\tools\PSpice\PSpice.ini file:
.lib "nom.lib" 

*Analysis directives: 
.TRAN  0 2n 0.1p 0.01p 
.PROBE V(alias(*)) I(alias(*)) W(alias(*)) D(alias(*)) NOISE(alias(*)) 
.INC "..\SCHEMATIC1.net" 


.END
