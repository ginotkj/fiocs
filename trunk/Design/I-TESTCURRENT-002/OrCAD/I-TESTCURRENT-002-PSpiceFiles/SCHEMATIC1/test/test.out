
**** 04/07/10 16:40:00 ******* PSpice 16.0.0 (July 2006) ****** ID# 0 ********

 ** Profile: "SCHEMATIC1-test"  [ D:\Documents\TESIS\fiocs\Design\I-TESTCURRENT-002\OrCAD\i-testcurrent-002-pspicefiles\schematic1\te


 ****     CIRCUIT DESCRIPTION


******************************************************************************




** Creating circuit file "test.cir" 
** WARNING: THIS AUTOMATICALLY GENERATED FILE MAY BE OVERWRITTEN BY SUBSEQUENT SIMULATIONS

*Libraries: 
* Profile Libraries :
* Local Libraries :
* From [PSPICE NETLIST] section of C:\OrCAD\OrCAD_16.0\tools\PSpice\PSpice.ini file:
.lib "nom.lib" 

*Analysis directives: 
.TRAN  0 2n 0.1p 0.01p 
.PROBE V(alias(*)) I(alias(*)) W(alias(*)) D(alias(*)) NOISE(alias(*)) 
.INC "..\SCHEMATIC1.net" 



**** INCLUDING SCHEMATIC1.net ****
* source I-TESTCURRENT-002
I_I1         N14389 0 DC 0Adc AC 0Aac
+EXP 0m 1m 500p 15p 510p 500p
R_R1         N14389 N14398  1k TC=0,0 
R_R2         0 N14398  1k TC=0,0 

**** RESUMING test.cir ****
.END

**** 04/07/10 16:40:00 ******* PSpice 16.0.0 (July 2006) ****** ID# 0 ********

 ** Profile: "SCHEMATIC1-test"  [ D:\Documents\TESIS\fiocs\Design\I-TESTCURRENT-002\OrCAD\i-testcurrent-002-pspicefiles\schematic1\te


 ****     INITIAL TRANSIENT SOLUTION       TEMPERATURE =   27.000 DEG C


******************************************************************************



 NODE   VOLTAGE     NODE   VOLTAGE     NODE   VOLTAGE     NODE   VOLTAGE


(N14389)    0.0000 (N14398)    0.0000 




    VOLTAGE SOURCE CURRENTS
    NAME         CURRENT


    TOTAL POWER DISSIPATION   0.00E+00  WATTS



          JOB CONCLUDED

**** 04/07/10 16:40:00 ******* PSpice 16.0.0 (July 2006) ****** ID# 0 ********

 ** Profile: "SCHEMATIC1-test"  [ D:\Documents\TESIS\fiocs\Design\I-TESTCURRENT-002\OrCAD\i-testcurrent-002-pspicefiles\schematic1\te


 ****     JOB STATISTICS SUMMARY


******************************************************************************



  Total job time (using Solver 1)   =       10.53
